packages/trpc/server/routers/viewer/teams/getUpgradeable.handler.ts

```diff
@@ -12,14 +12,32 @@ type GetUpgradeableOptions = {
 
 export const getUpgradeableHandler = async ({ ctx }: GetUpgradeableOptions) => {
   if (!IS_TEAM_BILLING_ENABLED) return [];
-  let { teams } = await prisma.user.findUniqueOrThrow({
-    where: { id: ctx.user.id },
-    include: { teams: { where: { role: MembershipRole.OWNER }, include: { team: true } } },
+
+  // Get all teams/orgs where the user is an owner
+  let teams = await prisma.membership.findMany({
+    where: {
+      user: {
+        id: ctx.user.id,
+      },
+      role: MembershipRole.OWNER,
+      team: {
+        parentId: null, // Since ORGS relay on their parent's subscription, we don't need to return them
+      },
+    },
+    include: {
+      team: {
+        include: {
+          children: true,
+        },
+      },
+    },
```

Comment:
Refactored query to make it more readable. We didnt really need to query the user table for this when the membership table had everything we need